### ISSUE TITLE
nltk.pos_tag performance

### ISSUE THREAD

**Issue by stevenbird** [nltk/nltk#1110]
NLTK's built-in POS tagger doesn't perform so well:

```
>>> text = word_tokenize("The quick brown fox jumps over the lazy dog")
>>> pos_tag(text)
[('The', 'DT'), ('quick', 'NN'), ('brown', 'NN'), ('fox', 'NN'), ('jumps', 'NNS'), ('over', 'IN'), ('the', 'DT'), ('lazy', 'NN'), ('dog', 'NN')]
```

We need a better built-in model.

cf. http://stackoverflow.com/questions/30821188/python-nltk-pos-tag-not-returning-the-correct-part-of-speech-tag

**Comment by ewan-klein** [comment]
@stevenbird Can we get one or more new labels for issues that try to improve coverage, accuracy, robustness of existing modules? I'm thinking also about e.g., better trained models for the dependency parser and the NER module, and making the CCG parser probabilistic.

**Comment by alvations** [comment]
@stevenbird , I think the simplest to "reimplement" with top quality accuracy is actually [honnibal's perception tagger](https://honnibal.wordpress.com/2013/09/11/a-good-part-of-speechpos-tagger-in-about-200-lines-of-python/). 

Next, the simplest to implement with unknown accuracy is ngram taggers with backoff ngram taggers using existing tagger corpora in NLTK or redistributable model that one can build with LDC's corpora (something like [Spaghetti tagger](https://code.google.com/p/spaghetti-tagger/).

There's one hand-crafted tagger with unknown accuracy too in `nltk.parse.malt`. Other rule-based tagger to consider is from [CLIPS' pattern.en](https://github.com/clips/pattern/blob/master/pattern/text/en/__init__.py#L169). The major caveats of pattern's tagger is the dependence on lexicon: https://raw.githubusercontent.com/clips/pattern/master/pattern/text/en/en-lexicon.txt

My last suggestion is a pre-trained Brill tagger, pattern has a list of rules: https://github.com/clips/pattern/blob/master/pattern/text/en/en-context.txt

**Comment by stevenbird** [comment]
@alvations: thanks for those suggestions. It looks like Honnibal's tagger uses an MIT license. @syllog1sm is there any objection to including this tagger in NLTK?

**Comment by honnibal** [comment]
Sure, go ahead. You might want my transition-based parser too, which is also under MIT I think.

**Comment by stevenbird** [comment]
@honnibal – great, thanks very much

**Comment by stevenbird** [comment]
Another example of bad behaviour: http://stackoverflow.com/questions/32571486/nltk-red-not-recognised-as-an-adjective

**Comment by stevenbird** [comment]
Resolved by #1122
